### DISCUSSION TITLE
multiorg

### DISCUSSION THREAD

**Discussion by bjensen54** [getredash/redash#6002] [General]
I had tried to add multiorg functionality by adding the environment variable REDASH_MULTI_ORG=true to the env file in the root redash folder.  Upon restarting all I could get was a 404 error when trying to hit the site.  Am I missing some other key piece of the configuration?  Please let me know.  This is on current v10 that was upgraded from v8 on the AMI.

Thank you - 

Brad

**Comment by manan19** [comment]
@bjensen54 Were you able to figure out to add the multi-org functionality? 

My guess is that if you instantiated redash with single org then it doesn't automatically migrate the database to multi-org.

**Comment by gpspake** [comment]
I started a separate discussion here and linked this one. My question is basically the same but I added some links to other threads I could find about multi org https://github.com/getredash/redash/discussions/6839
